### DISCUSSION TITLE
Search plugin records

### DISCUSSION THREAD

**Discussion by bogdancordos** [netbox-community/netbox#7711] [Q&A] [answered]
Hi,

Is the global "search" function able to look into plugin records?
Example:
I have installed a DNS plugin that holds nameservers, zones and records. I will be useful to search from those records from the global search.
I'm not sure if this is a NetBox functionality or the plugin needs to be implemented differently to accommodate this.

**Comment by jeremystretch** [comment, marked as answer]
> Is the global "search" function able to look into plugin records?

Unfortunately no; the models it queries are all hard-coded. We could expose this via the plugins API but honestly the whole global search function really needs to be redone first.
